File: Algorithms/Java/Helper/ArrayGenerator.java
import java.util.Random;

/**
 * @description:
 * @author: WenRuo
 * @date: 2021/10/22 11:25
 */

public class ArrayGenerator {

    private ArrayGenerator() {
    }

    /**
     * 生成一个有序数组
     *
     * @param n 传入生成数组大小
     * @return 一个有序数组
     */
    public static Integer[] generateOrderedArray(int n) {
        Integer[] arr = new Integer[n];
        for (int i = 0; i < n; i++)
            arr[i] = i;
        return arr;
    }

    /**
     * 生成一个随机数组
     *
     * @param n     数组的大小
     * @param bound 数组最大边界
     * @return 一个随机数组
     */
    public static Integer[] generateRandomArray(int n, int bound) {
        Integer[] arr = new Integer[n];
        Random rnd = new Random();
        for (int i = 0; i < n; i++)
            arr[i] = rnd.nextInt(bound);
        return arr;
    }

}

File: Algorithms/C#/Sort/ArrayGenerator.cs
//
// @Description:
// @Author: WenRuo
//


using System;

namespace Sort
{
    public class ArrayGenerator
    {
        private ArrayGenerator()
        {
        }

        /// <summary>
        /// 生成一个有序数组
        /// </summary>
        /// <param name="n"></param>
        /// <returns></returns>
        public static Int32[] GenerateOrderedArray(int n)
        {
            Int32[] arr = new Int32[n];
            for (int i = 0; i < n; i++)
                arr[i] = i;
            return arr;
        }

        /// <summary>
        /// 生成一个随机数组
        /// </summary>
        /// <param name="n"></param>
        /// <param name="bound"></param>
        /// <returns></returns>
        public static Int32[] GenerateRandomArray(int n, int bound)
        {
            Int32[] arr = new Int32[n];
            Random rnd = new Random();
            for (int i = 0; i < n; i++)
                arr[i] = rnd.Next(bound);
            return arr;
        }
    }
}